lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -657,16 +657,18 @@ MITIGATION_02_OPTIONS_WITH_F2F:
     pageId: pyi-suggest-other-options
   events:
     f2f:
-      targetState: CRI_F2F
+      targetState: MITIGATION_02_PYI_POST_OFFICE
```

Comment:
@rpayal The figma seems to suggest that this should still go directly to `CRI_F2F` - but not sure if that's correct?

Reply:
yeah correct, done as per figma.